scripts/bias_scan.py

```diff
@@ -61,16 +61,18 @@ def main(steps,ip_address):
         evl_bias=[]
         ecurrent=[]
         bias_value = hpk_value if ch in hpk else fbk_value
```

Comment:
Issue that is separate to this PR: several of these list variables don't get used.